app/src/main/java/xyz/jhughes/laundry/notificationhelpers/NotificationCreator.java

```diff
@@ -58,6 +58,10 @@ public void onFinish() {
     }
 
     private static void updateTimeNotification(String machine, Context context, long timeLeft) {
+        if (notifcationIds == null || self == null) {
```

Comment:
What error is this fixing? I mean probably a NPE, but do you know when it happens?